dotnet/BundleContents.md

```diff
@@ -276,6 +279,29 @@ The easiest way is to set `PublishFolderType` to `None`:
 <EmbeddedResource Include="MyFile.bin" CopyToOutputDirectory="PreserveNewest" PublishFolderType="None" />
 ```
 
+<a name="transitiveProject" />
+
+### `PublishFolderType` doesn't work from a referenced library project.
+
+The `PublishFolderType` metadata does not work in transitive / referenced
+projects that aren't platform-specific. For example: if an executable project
+references a `net9.0` library project, adding the `PublishFolderType` metadata
+on items in the `net9.0` library project doesn't have any effect, and you'll get a warning like this:
+
+> Xamarin.Shared.Sdk.targets(1836,3): Warning  : The file '/myresource/file.psd' does not specify a 'PublishFolderType' metadata, and a default value could not be calculated. The file will not be copied to the app bundle.
+
+The workaround is to set
+`MSBuildDisableGetCopyToPublishDirectoryItemsOptimization=true` in the
+executable project:
```

Comment:
If I remember correctly, the property should be set for the library (not executable) project. For the executable project it is already set out-of-the-box (somewhere in the xamarin sdk targets).

Reply:
Ah true, good catch, thanks!